test/desktop.spec.js

```diff
@@ -76,12 +76,12 @@ describe('Using monkey on desktop', function () {
       });
     });
 
-    it('should not make hyphens in names clickable', function () {
-      $container.find('.nonclickable.special-char .char:contains(-)').length.should.equal(1);
+    it('should not include hyphens in names', function () {
+      $container.find('.char:contains(-)').length.should.equal(0);
```

Comment:
Alright... I'm adding them back on the gem, then I'll revert this change. I removed that because the book preview was not working for some names with special chars.